test/court/court-config.js

```diff
@@ -46,41 +81,176 @@ contract('Court config', () => {
-      } = await courtHelper.getCourtConfig(1)
-
-      assert.equal(_feeToken, feeToken.address, 'Fee token does not match')
-      assertBn(_jurorFee, jurorFee, 'Juror fee does not match')
-      assertBn(_heartbeatFee, heartbeatFee, 'Heartbeat fee does not match')
-      assertBn(_draftFee, draftFee, 'Draft fee does not match')
-      assertBn(_settleFee, settleFee, 'Settle fee does not match')
-      assertBn(_commitTerms, commitTerms, 'Commit terms number does not match')
-      assertBn(_revealTerms, revealTerms, 'Reveal terms number does not match')
-      assertBn(_appealTerms, appealTerms, 'Appeal terms number does not match')
-      assertBn(_appealConfirmTerms, appealConfirmTerms, 'Appeal confirmation terms number does not match')
-      assertBn(_penaltyPct, penaltyPct, 'Penalty permyriad does not match')
-      assertBn(_finalRoundReduction, finalRoundReduction, 'Final round reduction does not match')
-      assertBn(_firstRoundJurorsNumber, firstRoundJurorsNumber, 'First round jurors number does not match')
-      assertBn(_appealStepFactor, appealStepFactor, 'Appeal step factor does not match')
-      assertBn(_maxRegularAppealRounds, maxRegularAppealRounds, 'Number af max regular appeal rounds does not match')
-      assertBn(_appealCollateralFactor, appealCollateralFactor, 'Appeal collateral factor does not match')
-      assertBn(_appealConfirmCollateralFactor, appealConfirmCollateralFactor, 'Appeal confirmation collateral factor does not match')
+      await courtHelper.checkConfig(0, initialConfig)
+    })
+  })
+
+  context('changes after init', () => {
+    beforeEach('move forward', async () => {
+      // move away from term zero
+      await courtHelper.setTerm(1)
+    })
+
+    it('fails setting config if no governor', async () => {
+      const from = sender
+      const configChangeTermId = 3
+
+      // make sure account used is not governor
+      assert.notEqual(courtHelper.governor, from, 'it is actually governor!')
+
+      const { promise } = await courtHelper.changeConfigPromise(originalConfig, configChangeTermId, sender)
+      await assertRevert(promise, ERROR_BAD_SENDER)
+    })
+
+    it('fails setting config in the past', async () => {
+      const configChangeTermId = 3
+      // move forward
+      await courtHelper.setTerm(configChangeTermId + 1)
+
+      const { promise } = await courtHelper.changeConfigPromise(originalConfig, configChangeTermId, courtHelper.governor)
+      await assertRevert(promise, ERROR_TOO_OLD_TERM)
+    })
+
+    it('fails setting config with only one term in advance', async () => {
+      const configChangeTermId = 3
+      // move forward
+      await courtHelper.setTerm(configChangeTermId - 1)
+
+      const { promise } = await courtHelper.changeConfigPromise(originalConfig, configChangeTermId, courtHelper.governor)
+      await assertRevert(promise, ERROR_TOO_OLD_TERM)
+    })
+
+    context('governor can change config', () => {
```

Comment:
I'd wrap the tests above in a `context('when the config change fails')` block and wrap the following tests into

```suggestion
    context('when the config change succeeds', () => {
```